gems/pending/appliance_console/database_replication_primary.rb

```diff
@@ -0,0 +1,41 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'appliance_console/database_configuration'
+require 'appliance_console/database_replication'
+
+module ApplianceConsole
+  class DatabaseReplicationPrimary < DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REGISTER_CMD = 'repmgr master register'.freeze
+
+    def initialize
+      self.cluster_name      = nil
+      self.node_number       = nil
+      self.database_name     = "vmdb_production"
+      self.database_user     = "root"
+      self.database_password = nil
+      self.primary_host      = LinuxAdmin::NetworkInterface.new(NETWORK_INTERFACE).address
+    end
+
+    def ask_questions
+      clear_screen
+      say("Establish Primary Replication Server\n")
+      ask_for_unique_cluster_node_number
+      ask_for_database_credentials
+      ask_continue_if_already_configured? &&
```

Comment:
After https://github.com/ManageIQ/manageiq/pull/9585/files#r73604592 this will make a lot more sense as:
`return false if repmgr_configured? && !confirm_reconfiguration`